src/content/getusermedia/getdisplaymedia/index.html

```diff
@@ -35,7 +35,12 @@ <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC
 
     <video id="gum-local" autoplay playsinline muted></video>
     <button id="startButton" disabled>Start</button>
-
+    <select id="preference" style="display:none">
+      <option value="" selected>Share the default</option>
+      <option value="browser">Share a browser window</option>
+      <option value="window">Share another window</option>
```

Comment:
```suggestion
      <option value="window">Share a browser window</option>
```

Reply:
Why "browser window"? It can be any native window.

Reply:
("Another" is also wrong, btw; one can share the current window too.)

Reply:
Good catch. I've edited my comment to "Share a window"